### PR TITLE
refactored to not be terrible anymore. could still use love in other …

### PR DESCRIPTION
**Pull request by elialbert**
…aspects

### CODE REVIEW

File: arrowhead/arrowhead.rb [hunk 1]
```diff
@@ -15,20 +15,27 @@ class Arrowhead
...
+      CLASSIFICATIONS[region] 
     else
       raise "Unknown region, please provide a valid region."
     end
   end
+
+  def self.get_shape(region_shapes, shape)
+    if region_shapes.include? shape
+      arrowhead = region_shapes[shape]
+      "You have a(n) '#{arrowhead}' arrowhead. Probably priceless."
```
Comment (DejaAugustine) [inline arrowhead/arrowhead.rb]: As an exercise, make it display either `a` or `an` depending on the type of arrowhead

**Comment by DejaAugustine** [comment]
Overall this looks good. Once you fix the HoundCI errors (you can use rubocop to check locally), I'll be ready to give this one a pass.

**Comment by elialbert** [comment]
@DejaAugustine 
OK updated

**Comment by DejaAugustine** [comment]
@elialbert looks good. :+1:

**Comment by paulghaddad** [comment]
@elialbert Nice work! You can move on.

**Pull request closed without merging**